src/githubcity/ghcity.py

```diff
@@ -540,7 +543,13 @@ def __readAPI(self, url):
                 self.__logger.exception("_readAPI: waiting 10 secs")
                 time.sleep(10)
 
-        data = json.loads(response.read().decode('utf-8'))
+        response_body = response.read()
+
+        if response.getheader('Content-Encoding') == 'gzip':
```

Comment:
Could you read also the response when the `Content-Encoding` is not gzip?
Only to be totally sure that we are not going to lose the responses.

Reply:
The response is read, then if there is a header that's `Content-Encoding: gzip`, it's decoded. Then either way the response is decoded as `utf-8`.

Reply:
Ops, sorry! I didn't understand properly the first time. You did properly. Thanks!